.eslintrc.cjs

```diff
@@ -0,0 +1,41 @@
+module.exports = {
```

Comment:
good question! i believe the recommendation by eslint team is to move to the new configuration file naming conventions

.eslintrc is still still supported but we just want to start using their suggested pattern because they are looking to deprecate in the future: https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats